### DISCUSSION TITLE
some questions about results

### DISCUSSION THREAD

**Discussion by zh2417** [luancarvalhomartins/PyAutoFEP#122] [Q&A]
I also meet this error about .xtc files
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/7fa05408-d65c-40dd-b211-d64aca882383)
But this problem does not affect the calculation of the final result.
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/12ec7984-9be4-4445-8e74-be67b57691fd)

Should I pay attention to this problem or fix it, or can I ignore this error?
Also, the results I calculated are slightly different from the results you gave. May I ask how is this going
my results：
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/52eac3c1-8595-40a3-aa55-abbf64169c1e)
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/a1823c50-5d3c-411f-8377-4e30d6d36c6d)
tutorial01‘s results：
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/26644e18-ab74-40d9-8705-770b37532d23)
![image](https://github.com/luancarvalhomartins/PyAutoFEP/assets/72326396/5b645067-5410-44ac-afc6-ddbc6d136801)
Thank you very much for your work and look forward to your reply.

**Comment by luancarvalhomartins** [comment]
Thank you very much for your question.

1. Currently, you can safe ignore that `Cannot stat`  error. I will write a fix for it. I should have, actually.
2. The results are actually quite similar, which is in line with our published and observed finding regarding FEP. This is completelly expected due to the nature of the MD sampling.
